wastewater/dataModel.WasteWaterTreatment/WasteWaterTreatment-schema.json

```diff
@@ -0,0 +1,760 @@
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "enum": [
+                        "Property"
+                    ]
+                },
+                "value": {
+                    "type": "number"
+                },
+                "unitCode": {
+                    "type": "string"
+                },
+                "observedAt": {
+                    "type": "string",
+                    "format": "date-time"
+                }
+            },
+            "required": [
+                "type",
+                "value"
+            ]
+        },
+        "pH": {
+            "type": "object",
+            "description": "Property. Water pH level measured in a tank",
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "enum": [
+                        "Property"
+                    ]
+                },
+                "value": {
+                    "type": "number"
+                },
+                "unitCode": {
+                    "type": "string"
+                },
+                "observedAt": {
+                    "type": "string",
+                    "format": "date-time"
+                }
+            },
+            "required": [
+                "type",
+                "value"
+            ]
+        },
+        "airflow": {
+            "type": "object",
+            "description": "Property. Airflow blown by a blower",
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "enum": [
+                        "Property"
+                    ]
+                },
+                "value": {
+                    "type": "number"
+                },
+                "unitCode": {
+                    "type": "string"
+                },
+                "observedAt": {
+                    "type": "string",
+                    "format": "date-time"
+                }
+            },
+            "required": [
+                "type",
+                "value"
+            ]
+        },
+        "energy": {
+            "type": "object",
+            "description": "Property. Energy consumed by a blower in kW",
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "enum": [
+                        "Property"
+                    ]
+                },
+                "value": {
+                    "type": "number"
+                },
+                "unitCode": {
+                    "type": "string"
+                },
+                "observedAt": {
+                    "type": "string",
+                    "format": "date-time"
+                }
+            },
+            "required": [
+                "type",
+                "value"
+            ]
+        },
+        "pressure": {

```

Comment:
Subproperties defined in the NGSI LD standard are not necessary (i.e. unitcode, observedAt, etc) it should be removed